javascript/javascript2/week1/spiritanimalg.js

```diff
@@ -0,0 +1,25 @@
+const spiritAnimal = [
+  "Confident Bear",
+  "Competitive Wolf",
+  "Introverted Panda",
+  "Graceful Butterfly",
+  "Powerful Horse",
+  "Mischivous Fox",
+  "Strongwilled Tiger",
+  "Couregues Lion",
+  "Loving Dolphin",
+  "Optimistic Hawk",
+];
+
+const inputName = document.getElementById("name");
+const input = document.getElementById("input");
+const randomName = function () {
+  if (inputName === "") {
+    input.innerHTML = "Please enter name here";
+  } else {
+    const randomNumber = Math.floor(Math.random() * spiritAnimal.length);
+    const randomSpiritAnimal = spiritAnimal[randomNumber];
+    input.innerHTML = `${inputName}` + `${randomSpiritAnimal}`;
```

Comment:
You are getting an error here and also your empty input check is not working because you are missing something. You are using the HTML element, but you need to access it. Read about all inputs properties here https://developer.mozilla.org/en-US/docs/Web/API/HTMLInputElement